python/xgrammar/kernels/__init__.py

```diff
@@ -1,5 +1,26 @@
 """The kernels for XGrammar."""
 
+import torch
+
 from .apply_token_bitmask_inplace_cpu import apply_token_bitmask_inplace_cpu
-from .apply_token_bitmask_inplace_cuda import apply_token_bitmask_inplace_cuda
-from .apply_token_bitmask_inplace_triton import apply_token_bitmask_inplace_triton
+
+apply_token_bitmask_inplace = {
+    "cpu": apply_token_bitmask_inplace_cpu,
+}
```

Comment:
This moves to a dictionary of the different kernel impls of this function. If cuda isn't available it's not added to then it's not added to the dictionary. These games for triton where if the optional dependency "triton" isn't installed, it's not added to the dictionary as a possible impl.